languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -36035,6 +36035,17 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TYPOGRAPHIE" name="Typographie" type="typographical">
+      <rulegroup id="PONCTUATION_SUR_ABREVIATIONS" name="Abréviations" default="temp_off">
+          <rule id="PONCT_CF" default="temp_off">
+              <pattern>
+                  <marker>
+                      <token>cf</token>
```

Comment:
I'd also write an AP or an exception for CF (club de futebol etc.) to avoid FPs.

Reply:
for the moment, I did not find a way to take this into account in the rule